test/e2e/framework/resource_usage_gatherer.go

```diff
@@ -556,3 +557,65 @@ func (g *ContainerResourceGatherer) StopAndSummarize(percentiles []int, constrai
 	}
 	return &summary, nil
 }
+
+// KubemarkResourceUsage is a struct for tracking the resource usage of kubemark.
+type KubemarkResourceUsage struct {
+	Name                    string
+	MemoryWorkingSetInBytes uint64
+	CPUUsageInCores         float64
+}
+
+func getMasterUsageByPrefix(prefix string) (string, error) {
+	sshResult, err := e2essh.SSH(fmt.Sprintf("ps ax -o %%cpu,rss,command | tail -n +2 | grep %v | sed 's/\\s+/ /g'", prefix), GetMasterHost()+":22", TestContext.Provider)
+	if err != nil {
+		return "", err
+	}
+	return sshResult.Stdout, nil
+}
+
+// getKubemarkMasterComponentsResourceUsage returns the resource usage of kubemark which contains multiple combinations of cpu and memory usage for each pod name.
```

Comment:
see https://github.com/kubernetes/kubernetes/pull/87271/files#diff-b498843cdcc70479a96c0cfffc6290afL44

This TODO from https://github.com/kubernetes/kubernetes/pull/38507/files#diff-b498843cdcc70479a96c0cfffc6290afR31 4 years ago.
GetKubemarkMasterComponentsResourceUsage  is  only used by https://github.com/kubernetes/kubernetes/pull/87271/files#diff-572985ec546140c5cf2bc24658132e83R187
so make it private.

For 
```
TODO: figure out how to move this to kubemark directory (need to factor test SSH out of e2e framework
```
I think that it is good time to re-examine and evaluate this TODO

if someone wishes to write a separate proposal for this work, please create an issue or TODO(@github) named to this function, We can easily track this down